packages/webview_flutter/webview_flutter_wkwebview/lib/src/v4/src/webkit_webview_controller.dart

```diff
@@ -35,54 +35,62 @@ class WebKitWebViewControllerCreationParams
   }) : this(webKitProxy: webKitProxy);
 
   final WKWebViewConfiguration _configuration;
+
+  /// Handles constructing objects and calling static methods for the WebKit
+  /// native library.
+  @visibleForTesting
+  final WebKitProxy webKitProxy;
 }
 
 /// An implementation of [PlatformWebViewController] with the WebKit api.
 class WebKitWebViewController extends PlatformWebViewController {
   /// Constructs a [WebKitWebViewController].
-  WebKitWebViewController(
-    PlatformWebViewControllerCreationParams params, {
-    @visibleForTesting WebKitProxy webKitProxy = const WebKitProxy(),
-  }) : super.implementation(params is WebKitWebViewControllerCreationParams
+  WebKitWebViewController(PlatformWebViewControllerCreationParams params)
+      : super.implementation(params is WebKitWebViewControllerCreationParams
```

Comment:
This removes the `webKitProxy` parameter from the object and just uses the one provided to the creation param.